requirements.txt

```diff
@@ -0,0 +1,2 @@
+numpy
+git+https://github.com/inducer/pytools.git#egg=pytools
```

Comment:
This is different from setup.py:

```
      install_requires=[
          "pytools>=2011.1",
          "pymbolic",
      ],
```

Reply:
It is. The mypy runner script I used doesn't use `setup.py`, otherwise it wouldn't be necessary. I personally don't care that much, but I'd be happy to see `requirements.txt` gone.

Reply:
Should we synchronize the requirements from setup.py and requirements.txt?